### PR TITLE
Update telephone numbers in eBenefits applications

### PR DESCRIPTION
**Pull request by micahchiang**
## Description
This pull request updates the VA311 phone number in ebenefits applications to 800-698-2411. This number update is cleared to go out on 10/30 per this discussion - https://dsva.slack.com/archives/C52CL1PKQ/p1603813162056500?thread_ts=1603718764.492500&cid=C52CL1PKQ. 

This pull request can wait on being merged until tomorrow (10/30), where it should be merged before the daily deploy goes out to production. 

## Testing done
- local, numbers updated

## Acceptance criteria
- [x] numbers updated

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by Mottie — APPROVED** [review]
ghe-emoji

File: src/applications/disability-benefits/686c-674/config/helpers.js [hunk 1]
```diff
@@ -52,10 +52,8 @@ export const ServerErrorAlert = (
     </p>
     <p>
       If you get this error again, please call the VA.gov help desk at{' '}
-      <a href="tel:8446982311" aria-label="8 4 4. 6 9 8. 2 3 1 1.">
-        844-698-2311
-      </a>{' '}
-      (TTY: <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
+      <Telephone contact="8006982411">800-698-2411</Telephone> (TTY:{' '}
```
Comment (sanlouise) [inline src/applications/disability-benefits/686c-674/config/helpers.js:55]: Would it make sense to add this phone number to CONTACTS in the Telephone component?

**Review by sanlouise — APPROVED** [review]

File: src/applications/disability-benefits/686c-674/config/helpers.js (hunk 1 above)
Comment (micahchiang) [reply src/applications/disability-benefits/686c-674/config/helpers.js:55]: Definitely worthwhile. I'll create a ticket for doing so 👍

Comment (sanlouise) [reply src/applications/disability-benefits/686c-674/config/helpers.js:55]: It appears Kelson already just pushed up [a PR](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/490/files) that does just this 😄

**Pull request merged**